### DISCUSSION TITLE
Input Capture Support

### DISCUSSION THREAD

**Discussion by sithlord48** [linuxdeepin/developer-center#10659] [Features Request & Ideas | 特性请求 & 头脑风暴]
Input capture is need to support applications like [input-leap](https://github.com/input-leap/input-leap) , and [deskflow](https://github.com/deskflow/deskflow) (formally synergy).

Support has added in the KDE and GNOME portals.
I am raising this issue in portals without support.  

The portal should also allow input capture access to the clipboard.

**Comment by BLumia** [comment]
Right now deepin does not really support Wayland since our Wayland compositor (and DDE portal as well) is still under development, hopefully we will support it when our Wayland session support is ready :)

**Reply by sithlord48** [reply]
This will also be needed for flatpaks of those applications
